openshift/app.dc.yaml

```diff
@@ -234,6 +234,16 @@ objects:
                     secretKeyRef:
                       key: password
                       name: "${APP_NAME}-objectstorage-secret"
+                - name: SERVER_ENCRYPTION_DB
+                  valueFrom:
+                    secretKeyRef:
+                      key: db
+                      name: "${APP_NAME}-encryption-keys"
```

Comment:
Here's another place that the db encryption stuff wasn't removed. It's also still in the secrets in the dev environment, so it should be removed from there to not cause confusion later on.

Reply:
done. instructions for secret do not reference `db` so new ones will not have `db` field.